addons/l10n_fr_certification/models/account_closure.py

```diff
@@ -0,0 +1,153 @@
+from datetime import datetime, timedelta
```

Comment:
this feature should be in a new module, since it's breaking our stable policy by adding new models and fields :-s